mlir/test/lit.cfg.py

```diff
@@ -126,6 +131,15 @@ def add_runtime(name):
 if config.enable_cuda_runner:
     tools.extend([add_runtime("mlir_cuda_runtime")])
 
+if config.mlir_run_arm_sme_tests:
+    config.substitutions.append(
+        (
+            "%arm_sme_abi_shlib",
```

Comment:
nit: `%arm_sme_runtime` or `%arm_sme_runtime_shlib`?

Reply:
I think "runtime" implies these are something more substantial, but this library should only ever contain a few SME ABI builtins.

Reply:
fair enough, that's what these are tho, runtime support routines [1]:
> Every platform that supports SME must provide the following runtime support routines:

[1] https://github.com/ARM-software/abi-aa/blob/main/aapcs64/aapcs64.rst#sme-support-routines

Reply:
I also think that "runtime" can be interpreted as something quite substantial. Having said that, I agree with Cullen that we should strive for consistency with the spec. I suggest "arm_sme_support_routines_shlib".

In any case, I wouldn't worry about this particular name too much :) (we can always rename later)